backend/src/main/resources/doc/openapi.json

```diff
@@ -1191,6 +1194,34 @@
                 "required": [
                     "type"
                 ]
+            },
+            "Badge": {
```

Comment:
This is the first part of LOON-403. We've discussed the API implementation with @rposkocil and came to the conclusion that we can amend our `/confirm` API within which we'll increase the points and add a badge (depending on the user's check up) to the user

Thus, by this change we'll also send badges the user has

@martin-ladecky @rposkocil WDYT?

Reply:
Too nice to be true :). If Jana does not see any problems, I am ok with it.

Reply:
I'll scheduler a call with Jana sometime this week and keep you updated here in the comments

Reply:
@martin-ladecky I just talked to Jana, and she confirmed that my understanding is correct, so we're good